scripts/interface.d.ts

```diff
@@ -0,0 +1,22 @@
+import { WNode } from '@dojo/framework/widget-core/interfaces';
```

Comment:
How come we need a shared interfaces file for the pipeline scripts?

Reply:
The rehype-prism library used previously did not have an option for loading the extra languages (like jsx / tsx) we needed. So I copied their file and added in the language loading piece. I will probably submit a PR back to the actual library but this was faster and I needed to test it anyways.

Reply:
Seemed easier from an organizational perspective to have them all in one place, but they can be split up if we want.